build.sh

```diff
@@ -336,6 +336,13 @@ if [ ${BUILD_FE} -eq 1 -o ${BUILD_SPARK_DPP} -eq 1 ]; then
         cp -r -p ${STARROCKS_HOME}/fe/spark-dpp/target/spark-dpp-*-jar-with-dependencies.jar ${STARROCKS_OUTPUT}/fe/spark-dpp/
         cp -r -p ${STARROCKS_THIRDPARTY}/installed/aliyun_oss_jars/* ${STARROCKS_OUTPUT}/fe/lib/
 
+        if [[ ! -f ${STARROCKS_OUTPUT}/broker_thirdparty_jars.tar.gz ]]; then
+            # download tencent hadoop cos jar, this can not be found in maven repository, so we download it manually
+            echo "download aliyun hadoop oss and tencent hadoop cos jar..."
+            wget http://cdn-thirdparty.starrocks.com/broker_thirdparty_jars.tar.gz -O ${STARROCKS_OUTPUT}/broker_thirdparty_jars.tar.gz
```

Comment:
Can it be consistent with other third-party jars and integrated into docker image.